MiroApp.ts

```diff
@@ -115,4 +117,13 @@ export class MiroApp extends App {
             .getByUsername(this.botUsername)) as IUser;
         return true;
     }
+
+    public async executeViewSubmitHandler(context: UIKitViewSubmitInteractionContext, read: IRead, http: IHttp, persistence: IPersistence, modify: IModify) {
+        const handler = new ExecuteViewSubmitHandler(this, read, http, modify, persistence);
+        return await handler.run(this, context, read, http, persistence, modify);
```

Comment:
Why pass all the params on the constructor and in the method? If you're already receiving these `(this, read, http, modify, persistence)` on the constructor, there's no need to pass again in the method call. You can pass only the `context`.

Reply:
Adding changes in next PR